include/exadg/grid/grid.cpp

```diff
@@ -160,10 +160,50 @@ Grid<dim>::create_triangulation(
     {
       // we need to generate the coarse triangulations explicitly
 
-      // TODO
-      // construct all the coarser triangulations
+      // resize the empty coarse triangulations vector
+      coarse_triangulations = std::vector<std::shared_ptr<dealii::Triangulation<dim> const>>(
+        triangulation->n_global_levels());
+
+      unsigned int coarse_level = triangulation->n_global_levels() - 1;
+
+      // lambda function for creating and placing the coarse triangulations
+      auto const lambda_create_coarse_levels = [&](unsigned int              global_refinement,
+                                                   std::vector<unsigned int> refine_local) {
+        auto new_tria = std::make_shared<dealii::parallel::fullydistributed::Triangulation<dim>>(
```

Comment:
I would then use `level_tria` here, because `new` is very general and can mean a lot.

Reply:
I agree. I wanted to keep our naming similar to dealii. See the equivalent function in dealii using the variable name `new_tria`: https://github.com/dealii/dealii/blob/07b5a85db9d97aac577b56873c6c34ff4f3bf73a/include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h#L2439-L2451

Reply:
changed in fe49537a6dd436ad6b49130feb488cd584e5d308.